specs/phase1/beacon-chain.md

```diff
@@ -111,7 +119,7 @@ The following values are (non-configurable) constants used throughout the specif
 
 | Name | Value |
 | - | - |
-| `PHASE1_BASE_REWARDS_PER_EPOCH` | `uint64(6)` |
+| `PHASE1_BASE_REWARDS_PER_EPOCH` | `uint64(5)` |
```

Comment:
Base rewards reduced to 5 because the two timeliness rewards sum to 1 base reward